pkg/common/fileutils/fileutils.go

```diff
@@ -150,3 +150,10 @@ func ReadLineByLine(filePath string, processLine func(line []byte) error) error
 
 	return nil
 }
+
+func TrimLeadingSlash(s string) string {
+	if len(s) > 0 && s[0] == '/' {
+		return s[1:]
+	}
+	return s
+}
```

Comment:
When would we want to do this? Changing `/foo/bar/` to `foo/bar/` seems odd to me since it changes where the file is